frontend/src/component/common/ProFeatureTooltip/ProFeatureTooltip.tsx

```diff
@@ -29,17 +26,43 @@ const StyledLink = styled(Link)(({ theme }) => ({
     width: 'fit-content',
 }));
 
-export const ProFeatureTooltip = ({ children }: ProFeatureTooltipProps) => {
+export interface ProFeatureTooltipProps {
+    children: React.ReactNode;
+    title: string;
+    origin: string;
+    center?: boolean;
+}
+
+export const ProFeatureTooltip = ({
```

Comment:
What do you think about renaming this component to `PremiumFeatureTooltip` since it covers both Pro + Enterprise and Enterprise?

Reply:
It seems like it's not even a `Tooltip` sometimes now, so even more of a reason to change it I think 🤔

Reply:
makes sense yes, I will think of a better name